akka-docs/src/test/java/jdocs/stream/javadsl/cookbook/RecipeMissedTicks.java

```diff
@@ -76,9 +74,10 @@ public void work() throws Exception {
         pub.sendNext(Tick);
         pub.sendNext(Tick);
 
-        FiniteDuration timeout = FiniteDuration.create(200, TimeUnit.MILLISECONDS);
+        scala.concurrent.duration.FiniteDuration timeout =
+                scala.concurrent.duration.FiniteDuration.create(200, TimeUnit.MILLISECONDS);
 
-        Await.ready(latch, Duration.create(1, TimeUnit.SECONDS));
+        Await.ready(latch, scala.concurrent.duration.Duration.create(1, TimeUnit.SECONDS));
```

Comment:
Not shown in sample, can be left as it was

Reply:
Nice to be able to remove the `scala.concurrent.duration` from the file-wide imports though, to avoid accidentally using them in the future.